api/src/main/scala/dcos/metronome/api/ApiModule.scala

```diff
@@ -16,46 +17,37 @@ import mesosphere.marathon.core.plugin.PluginManager
 import mesosphere.marathon.metrics.Metrics
 import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
 import play.api.http.HttpErrorHandler
-import play.api.mvc.{ AnyContent, BodyParser }
+import play.api.mvc.ControllerComponents
 import play.api.routing.Router
 import router.Routes
 
 import scala.concurrent.ExecutionContext
 
-class ApiModule(
+class ApiModule(controllerComponents: ControllerComponents, assets: Assets, httpErrorHandler: HttpErrorHandler)(
+  implicit
```

Comment:
Why `ec` is made to be the very first argument, as it seems, everywhere?

Reply:
No reason in particular. Ve was already implicit so I just added the other ones after. 

I can change it but my question as well is if we should keep everything as implicit or not. I made the change but I’m not sure if it is any better

Reply:
I don't see a reason to change it. + the diff would be smaller (less reformatting)

Reply:
I’ll rollback this update the

Reply:
Actually, I think read your comment wrong. You prefer keeping like this or revert and not have ec as the first one?

Reply:
As far as I know, by putting `implicit ec: ...` first, the rest of parameters are made implicit too. Having as few implicts as possible makes the code more readable. What do you think?

Reply:
Meaning, `ec` should be placed as it is on master.

Reply:
Agreed, we're conservative with implicits round these parts :)